detect-nuget-inspector/detect-nuget-inspector/Inspection/Inspectors/SolutionInspector.cs

```diff
@@ -129,14 +129,14 @@ public Container GetContainer()
                                 bool fileNotFound = true;
                                 while (fileNotFound)
                                 {
-                                    parentPath =
-                                        parentPath.Substring(0, OperatingSystem.IsWindows() ? parentPath.LastIndexOf("\\") : parentPath.LastIndexOf("/"));
                                     string checkFile = Path.Combine(parentPath, projectRelativePath);
                                     if (parentPath.Equals(Path.GetPathRoot(solutionDirectory)) || parentPath.Equals(""))
```

Comment:
So in the problem case was parentPath the empty string and the equals("") check will protect us now that it is higher in the execution? Is it worth checking for null earlier in the if before calling the .Equals?

Reply:
Yeah, I believe that was the problem. It was not happening on Linux and Mac for some reason. Yeah that might be worth checking, will update it in the next commit.